ProjectShop/OrderItems/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+from order.models import Order
```

Comment:
According to PEP 8:
Imports should be grouped in the following order:
1. Standard library imports.
2. Related third-party imports.
3. Local application/library-specific imports.
You should put a blank line between each group of imports.